Testing/functional/e2e/cypress/integration/timeline/modals.js

```diff
@@ -5,15 +5,22 @@ describe('Validate Modals Popup', () => {
         cy.login(Cypress.env('keycloak.username'), Cypress.env('keycloak.password'), AuthMethod.KeyCloak)
```

Comment:
Should we call this test 'Dismiss modal by View Result' instead of 'Covid Modal'?